src/main/java/it/gov/pagopa/rtd/ms/rtdmsfileregister/controller/RestControllerImpl.java

```diff
@@ -0,0 +1,42 @@
+package it.gov.pagopa.rtd.ms.rtdmsfileregister.controller;
+
+import it.gov.pagopa.rtd.ms.rtdmsfileregister.model.FileMetadataDTO;
+import java.math.BigDecimal;
+import java.util.List;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@ResponseBody
+@Slf4j
+public class RestControllerImpl implements
+    it.gov.pagopa.rtd.ms.rtdmsfileregister.controller.RestController {
+
+  @Override
+  public void healthCheck() {
+    //Return OK if the service is reachable
+  }
+
+  @Override
+  public List<FileMetadataDTO> getFileStatus() {
+    FileMetadataDTO stub = new FileMetadataDTO("test0", "abc", 5, 2, new BigDecimal(900),
+        new BigDecimal(700), 5, 0);
+    return List.of(stub);
+  }
+
+  @Override
+  public ResponseEntity<String> setFileStatus(@NotNull @NotBlank FileMetadataDTO body) {
```

Comment:
@TommasoLencioni In my opinion it's better to use annotations (like NotNull or NotBlank) on interface than implementation. Moreover, in the controller implementation, we should prefer to return plain objects not http ResponseEntities. A good example is the method getFileStatus, or you can find a more complex one here
https://github.com/pagopa/fa-ms-customer/blob/release-dev/api/rest/src/main/java/it/gov/pagopa/fa/customer/controller/FaCustomerControllerImpl.java